draft-ietf-jsonpath-base.md

```diff
@@ -689,7 +687,7 @@ objects, only arrays, or both.
 The resulting nodelist of a child selector is the concatenation of
 the nodelists from each of its //PICKERS// in the order that the //PICKERS//
 appear in the list.
-Note that any node selected in more than one of the selector entries is kept
+Note that any node selected by more than one //PICKER// is kept
 as many times in the nodelist.
```

Comment:
Saying that a //PICKER// selects something confuses //PICKERS// and selectors. How about this?
```suggestion
Note that any node matched by more than one //PICKER// is kept
as many times in the nodelist.
```

Reply:
"Matches" (or maybe "identifies") is fine.